### PR TITLE
Add Metallothionein family

### PR DESCRIPTION
**Pull request by steppi**
This PR adds the Metallothionein family. https://www.genenames.org/data/genegroup/#!/group/638

### CODE REVIEW

File: entities.csv [hunk 1]
```diff
@@ -264,6 +264,7 @@ Mitochondrial_Ribosome
 MRPS
 MRPL
 MMP
+MT
```
Comment (bgyori) [inline entities.csv]: It might make sense to use Metallothionein as the name

Comment (steppi) [reply entities.csv]: OK. I went with the shortform because this is a very common grounding for MT, but the full name is more descriptive

Comment (bgyori) [reply entities.csv]: Yeah, MT might be okay too, @johnbachman what do you think?

Comment (steppi) [reply entities.csv]: image

Comment (steppi) [reply entities.csv]: it's a close second to microtubule

**Pull request closed without merging**